nservicebus/upgrades/acs-host-6to7.md

```diff
@@ -8,6 +8,42 @@ related:
 - nservicebus/upgrades/5to6
 ---
 
+## Namespace changes
+
+All types specified in this document are now in the `NServiceBus` namespace.
+
+## IConfigureThisEndpoint changes
+
+`IConfigureThisEndpoint.Customize` is passed an instance of `EndpointConfiguration` instead of `BusConfiguration`.
+
+snippet: AzureServiceBusTransportWithAzureHost
+
+## AsA_Host role changes into IConfigureThisHost
+
+The specifiers `IConfigureThisEndpoint` and `AsA_Host` are now merged into `IConfigureThisHost`. 
+
+snippet: AsAHost
+
+### Removal of DynamicHostControllerConfig
+
+The `DynamicHostControllerConfig`configuration section has been removed, instead the `IConfigureThisHost.Customize` implementation requires to return an instance of `HostSettings` which contains all the configuration values. 
```

Comment:
I don't know, it's the interface itself that matters more than the name of the method imo